htdocs/bootstrap/css/custom-css.css

```diff
@@ -133,17 +133,8 @@ Note: other colors are currenly inherited from Bootstrap default color palette
 /*
```

Comment:
@HenriRabalais I'm guessing these CSS changes were done for the biobank module and will be part of your Biobank Pr to loris ?